persistence/CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ All notable changes to this module will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+**TODO:** consolidate `persistence/docs/CHANGELOG` and `persistence/CHANGELOG.md`
```

Comment:
I don't think our automatic search handles the ** properly so lets remove it.

```suggestion
TODO: consolidate `persistence/docs/CHANGELOG` and `persistence/CHANGELOG.md`
```

Reply:
Sounds great! I will avoid ** in the future